zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -1374,8 +1377,10 @@ pub(crate) fn checkpoint_gaps<T: ShieldedPoolTester>() {
         account.birthday().height() + 10,
         BlockHash([0; 32]),
         &not_our_key,
-        AddressType::DefaultExternal,
-        not_our_value,
+        &[FakeCompactOutput {
+            address_type: AddressType::DefaultExternal,
+            value: not_our_value,
+        }],
```

Comment:
You defined a `FakeCompactOutput::new` function, but are not using it here:
```suggestion
        &[FakeCompactOutput::new(
            AddressType::DefaultExternal,
            not_our_value,
        )],
```

EDIT: Looks like this was fixed in the second commit. I'd prefer it to be done in the first commit if you're making other changes to that commit.